charts/fpl-case-service/values.yaml

```diff
@@ -50,3 +53,6 @@ java:
         - AppInsightsInstrumentationKey
         - scheduler-db-password
   applicationPort: 4000
+
+ccd:
```

Comment:
tags:
  fpl-ccd-stack: false 
in first line should turn off ccd dependency, so this maybe is not needed

Reply:
Agreed, so I will remove.